Example/GiniVision/AppDelegate.swift

```diff
@@ -38,6 +38,29 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
     
+    func application(_ application: UIApplication, open url: URL, sourceApplication: String?, annotation: Any) -> Bool {
+        // If the current root view controller is not the ScreenAPIViewController, it won't do anything.
+        guard let navVC = window?.rootViewController as? UINavigationController, let screenAPIVC = navVC.viewControllers.first as? ScreenAPIViewController else {
+            return false
+        }
+        
+        // 1. Create GiniConfiguration
+        let giniConfiguration = GiniConfiguration()
+        giniConfiguration.debugModeOn = true
+        giniConfiguration.navigationBarItemTintColor = UIColor.white
```

Comment:
I don't really like this. The Gini Configuration is typically created elsewhere so this code is duplicated.

And setting the `navigationBarItemTintColor` is kind of strange. Why would the Open With feature care about the tint color. Set it somewhere where it would apply for all cases.

Reply:
I just copied the code from the ScreenAPIViewController in order to have the same results. It is not even necessary to add any special attribute to GiniConfiguration, but It was just for having the same results.

Reply:
`I just copied the code` Exactly the problem. I rest my case.

Reply:
I don't understand. What is the problem here? Should I avoid duplicated code in this case?

Reply:
Yeah, find a way to not have this here. Apply the configuration once in a place where it would apply for all cases (Open With or normal app start).

Reply:
You could move this into the ScreenAPIViewController to a public method similar to `easyLaunchGiniVision` named maybe `easyLaunchGiniVision(withImportedFile:)` or something. It's better to have Screen API stuff in one class.

Also for the Component API you could add a similar method to `ComponentAPIAnalysisViewController` and allow clients to opt for trying out the Component API here. A simple enum property with a switch would look nice I think, just an idea: 
```
switch giniVisionAPI {
case .Screen:
    print("Screen")
case .Component:
    print("Component")
}
```

Reply:
I came up with the idea of showing an alert which allows to choose between ScreenAPI and ComponentAPI when a file is imported. I don't know if the title and message of the alert are correct (I translated it to german using GT).